modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -166,7 +166,8 @@ const setup = (editor: Editor): RenderInfo => {
     return OuterContainer.parts.header({
       dom: {
         tag: 'div',
-        classes: [ 'tox-editor-header' ],
+        classes: [ 'tox-editor-header' ]
+          .concat((hasMultipleToolbar || hasToolbar || hasMenubar) ? [] : [ 'tox-editor-header--empty' ]),
```

Comment:
This should probably be a separate variable declared before this to make sure it's clear what it's doing. It might also make sense to have this be a spread instead of a `concat` but we'd done both here so not sure.

- [ ] Move to separate variable

Reply:
I’m using `concat` since it is used in other similar cases for setting classes in the same file (for example [line 230](https://github.com/tinymce/tinymce/blob/7276d5a31e449ecdddafcb1ae324ec9cb572e251/modules/tinymce/src/themes/silver/main/ts/Render.ts#L230) or [line 313](https://github.com/tinymce/tinymce/blob/7276d5a31e449ecdddafcb1ae324ec9cb572e251/modules/tinymce/src/themes/silver/main/ts/Render.ts#L313).

Grouped this variable with `hasToolbar`, `hasMenubar` and the like.